eventGenerator/services/HNService.js

```diff
@@ -0,0 +1,55 @@
+var firebase = require('firebase/app');
+require('firebase/database');
+
+var config = {
+  databaseURL: 'https://hacker-news.firebaseio.com'
+};
+firebase.initializeApp(config);
+var version = '/v0';
+var api = firebase.database().ref(version);
+
+// https://firebase.google.com/support/guides/firebase-web
```

Comment:
Is it necessary comment? And why do you use `var`? `const` and `let` are supported by the latest node.js